ci/builders/local_engine.json

```diff
@@ -0,0 +1,151 @@
+{
+  "builds": [
+    {
+      "drone_dimensions": [
+        "os=Mac-13",
+        "os=Linux"
+      ],
+      "gn": [
+        "--runtime-mode",
+        "debug",
+          "--android",
+          "--android-cpu=arm64",
+          "--no-stripped",
+          "--enable-impeller-vulkan",
+          "--no-lto"
+      ],
+      "name": "android_debug_arm64",
+      "ninja": {
+          "config": "android_debug_arm64",
+          "targets": [
+              "flutter",
+              "flutter/shell/platform/android:abi_jars"
+          ]
+      }
+    },
+    {
+      "drone_dimensions": [
+        "os=Mac-13",
+        "os=Linux"
+      ],
+      "gn": [
+        "--runtime-mode",
+        "profile",
+          "--android",
+          "--android-cpu=arm64",
+          "--no-stripped",
+          "--enable-impeller-vulkan",
+          "--no-lto"
+      ],
+      "name": "android_profile_arm64",
+      "ninja": {
+          "config": "android_profile_arm64",
+          "targets": [
+              "flutter",
+              "flutter/shell/platform/android:abi_jars"
+          ]
+      }
+    },
+    {
+      "drone_dimensions": [
+        "os=Mac-13",
+        "os=Linux"
+      ],
+      "gn": [
+        "--runtime-mode",
+        "release",
+          "--android",
+          "--android-cpu=arm64",
+          "--no-stripped",
+          "--enable-impeller-vulkan",
+          "--no-lto"
+      ],
+      "name": "android_release_arm64",
+      "ninja": {
+          "config": "android_release_arm64",
+          "targets": [
+              "flutter",
+              "flutter/shell/platform/android:abi_jars"
+          ]
+      }
+    },
+    {
+      "drone_dimensions": [
+        "os=Mac-13",
+        "os=Linux"
+      ],
+      "gn": [
+          "--runtime-mode",
+          "debug",
+          "--no-stripped",
+          "--enable-impeller-vulkan",
+          "--no-lto"
+      ],
+      "name": "host_debug",
+      "ninja": {
+          "config": "host_debug",
+          "targets": [
+              "flutter:unittests",
+              "flutter/build/archives:artifacts",
+              "flutter/build/archives:dart_sdk_archive",
+              "flutter/build/archives:embedder",
+              "flutter/build/archives:flutter_patched_sdk",
```

Comment:
Do we need to run archive targets for local development builds?

Reply:
Good question. @zanderso ?

Reply:
Done.